editor/src/clj/editor/html.clj

```diff
@@ -44,5 +44,5 @@
     :search-fn search-fn
     :search-value-fn search-value-fn
     :node-type HtmlNode
-    :view-types [:html]
+    :view-types [:html :text]
     :view-opts nil))
```

Comment:
Maybe we should think about it as part of this task, but it's kinda related:
https://github.com/defold/defold/issues/8550

this one is about html (but probably with much more functions we curently want) https://github.com/defold/defold/issues/8126

Reply:
Yeah, it would be nice. But it is a lot more work to have it be editable, not just viewable.